### PR TITLE
[doc](mtmv)mtmv doc

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+-->
+
+# 物化视图
+
+在数据仓库环境中，应用程序通常会对大型表执行复杂的查询。
+
+一个例子是SELECT语句对包含数十亿行的表执行多表连接和聚合。就系统资源和计算结果所花费的时间而言，处理这些查询可能代价高昂。
+物化视图提供了一种解决这些问题的方法。
+物化视图包含基于对一个或多个基表的SQL查询而预先计算的结果集。
+您可以发出SELECT语句来查询物化视图，与查询数据库中的其他表或视图的方式相同。Doris从物化视图返回预先计算的结果，不需要访问基表。
```
Comment (wm1581066) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view.md:34]: 得这里也提下透明改写的能力

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+table相关的属性，例如副本数，仍通过`ALTER TABLE`相关的语法来修改
+
+### 查看已创建的物化视图
+
+物化视图独有的特性可以通过[mv_infos()](../sql-manual/sql-functions/table-functions/mv_infos.md)查看
+
+和table相关的属性，仍通过[SHOW TABLES](../sql-manual/sql-reference/Show-Statements/SHOW-TABLES.md)来查看
+
+### 手动刷新物化视图
+
```
Comment (wm1581066) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view.md:82]: 整个这部分得有连贯起来的完善的例子

**Review by wm1581066 — CHANGES_REQUESTED** [review]

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- 湖仓加速。
+
+  查询是在外部catalog上进行的(例如存在hive catalog)，与查询本地数据库表相比，这可能会有较低的性能，这时可以通过建立物化视图，把外部数据存在Doris，来加速查询。
+
+## 使用物化视图
+
+### 创建物化视图
+
+物化视图支持多种刷新策略，
```
Comment (morningman) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view.md:60]: 这句去掉

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+### 删除物化视图
+物化视图有专门的删除语法，不能通过drop table来删除，
+
+具体的语法可查看[DROP ASYNC MATERIALIZED VIEW](../sql-manual/sql-reference/Data-Definition-Statements/Drop/DROP-ASYNC-MATERIALIZED-VIEW.md)
+
+### 修改物化视图
+
+修改物化视图的名字，物化视图的刷新方式及物化视图特有的property可通过[ALTER ASYNC MATERIALIZED VIEW](../sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-ASYNC-MATERIALIZED-VIEW.md)来修改
+
+table相关的属性，例如副本数，仍通过`ALTER TABLE`相关的语法来修改
```
Comment (morningman) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view.md:73]:
```suggestion
物化视图本身也是一个 Table，所以 Table 相关的属性，例如副本数，仍通过`ALTER TABLE`相关的语法来修改。
```

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md (hunk 3 above)
Comment (morningman) [reply docs/zh-CN/docs/query-acceleration/async-materialized-view.md:60]: 然后每一个操作，给一个最基础的sql示例，并且各个示例直接尽量能够串起来

File: docs/zh-CN/docs/query-acceleration/async-materialized-view.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+- Ad Hoc
+  
+  如果查询结果只包含基表中的少数行，或者查询结果需要大量的逻辑处理，例如半结构数据或花费大量时间的聚合。这时如果基表的数据变更不频繁，可以用物化视图来加速查询。
+
+- 湖仓加速。
+
+  查询是在外部catalog上进行的(例如存在hive catalog)，与查询本地数据库表相比，这可能会有较低的性能，这时可以通过建立物化视图，把外部数据存在Doris，来加速查询。
+
+## 使用物化视图
```
Comment (morningman) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view.md:56]:
```suggestion
## 物化视图的构建和维护
```

后续对应一个：

```
## 物化视图的使用
```

增加查询和透明改写方面的文档

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Review by yiguolei — DISMISSED** [review]
lgtm

**Comment by zddr** [comment]
run feut

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by seawinde — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**